src/serde_helpers.rs

```diff
@@ -380,6 +254,285 @@ pub mod datetime {
     );
 }
 
+/// Type converters for serializing and deserializing `u32` using [`serde_with::serde_as`].
+///
+/// ## Available converters
+/// - [`u32::FromTimestamp`] — converts a [`crate::Timestamp`] to and from a `u32`.
+/// - [`u32::AsTimestamp`] — converts a `u32` to and from a [`crate::Timestamp`].
```

Comment:
Slightly pedantic design thought: since `Timestamp` is a type defined in this crate, it makes a bit more sense to me to have these within a `timestamp` module that we can include any future timestamp helpers in as well.